pkg/netutil/cidr.go

```diff
@@ -3,9 +3,37 @@ package netutil
 import (
 	"net"
 
+	"github.com/stackrox/rox/pkg/logging"
 	"github.com/stackrox/rox/pkg/utils"
 )
 
+var (
+	privateSubnets = []*net.IPNet{
+		MustParseCIDR("127.0.0.0/8"),    // IPv4 localhost
+		MustParseCIDR("10.0.0.0/8"),     // class A
+		MustParseCIDR("172.16.0.0/12"),  // class B
+		MustParseCIDR("192.168.0.0/16"), // class C
+		MustParseCIDR("::1/128"),        // IPv6 localhost
+		MustParseCIDR("fd00::/8"),       // IPv6 ULA
+	}
+	log = logging.LoggerForModule()
+)
+
+// IsCIDRBlockInPrivateSubnet parses cidrStr and checks if it falls under the RFC 1819 private IP range
+func IsCIDRBlockInPrivateSubnet(cidrStr string) bool {
```

Comment:
For a library function in `netutil`, it would be more natural to operate on a `*net.IPNet` rather than possibly duplicating the parsing effort.